components/service/glean/README.md

```diff
@@ -134,18 +134,24 @@ run-time, using the following [`adb`](https://developer.android.com/studio/comma
 
 `adb shell am start -n [package.name]/mozilla.components.service.glean.debug.GleanDebugActivity [extra keys]`
 
-Where the `[package.name]` is the product's package name as exported in the manifest file (e.g.
-`org.mozilla.samples.glean` for the glean sample application) and `[extra keys]` is a list of
-extra keys to be passed to the debug activity. See the [documentation](https://developer.android.com/studio/command-line/adb#IntentSpec)
-for the command line switches used to pass the extra keys. These are the currently supported keys:
-
-|key|type|description|
-|---|----|-----------|
-| logPings | boolean (--ez) | If set to `true`, glean dumps pings to logcat; defaults to `false` |
-| sendPing | string (--es) | Sends the ping with the given name immediately |
-| tagPings | string (--es) | Tags all outgoing pings as debug pings to make them available for real-time validation. The value must match the pattern `[a-zA-Z0-9-]{1,20}` |
-
-For example, to direct the glean sample application to (1) dump pings to logcat, (2) tag the ping 
+In the above:
+
+- `[package.name]` is the product's package name as exported in the manifest
```

Comment:
I think this is called "application id" in the [Android docs](https://developer.android.com/studio/build/application-id), so we can rename this to `[application.id]`.  By doing so, we can simply state that the full application id is required, including any suffix for the built flavour specified in the gradle build script.

Reply:
That makes sense.  I still like the idea of including what it is for the sample app here, though.